### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by gaboesquivel**
This PR solves #30, updates the core team list and adds info about the continuous integration process and automated code quality checks.

### CODE REVIEW

**Comment by gaboesquivel** [comment]
@kevin-wolf please check the info is accurate.

File: README.md [hunk 1]
```diff
@@ -149,10 +151,49 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 [docs/create-react-app.md](docs/create-react-app.md)
 
+** ÐMeetup App Components **
```
Comment (kevin-wolf) [inline README.md]: Not rendering properly.

File: README.md [hunk 2]
```diff
@@ -149,10 +151,49 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
...
+		│   ├── store
+		│   └── utils
+		├── index.js
+		├── meetups                   meetup specific components
+		│   ├── components
+		│   ├── constants
+		│   ├── models
+		│   └── routes
+		└── serviceWorker.js
+```
```
Comment (kevin-wolf) [inline README.md]: The code format is wrong. Also, please add more explanation:

```
.
├── docs/ .............................................. documentation files and media
├── public/ ............................................ static and public files
├── src/ ............................................... application source code (we will cover in detail soon)
├── CONTRIBUTING.md .................................... contributing guidelines
├── LICENSE ............................................ project license
├── README.md .......................................... project homepage
├── package.json ....................................... dependencies manifest
├── docker-compose.yaml ................................ docker compose for local smart contracts development
├── config-overrides.js ................................ configuration overrides for `create-react-app`
├── netlify.toml ....................................... netlify configuration file
├── .env ............................................... environment variables (for local development)
├── .eslintrc .......................................... code style rules
├── .travis.yml ........................................ travis ci configuration file
```

Comment (gaboesquivel) [reply README.md]: ok, didn't think detailing each file was necessary, but yeah, it doesn't hurt either. What about the CI section, is that accurate ?

File: README.md (hunk 1 above)
Comment (gaboesquivel) [reply README.md]: oh, I think it was a dumb atomo tab config plugin I installed for php scripting, will fix. ( and also write php with 2 space tabbing!! )

File: README.md (hunk 2 above)
Comment (kevin-wolf) [reply README.md]: @gaboesquivel yeah I think more detail wouldn't hurt anybody :P
Yes, CI is fine.

**Comment by gaboesquivel** [comment]
@kevin-wolf I just made those corrections

**Pull request merged**